packages/builder/src/components/integration/RestQueryViewer.svelte

```diff
@@ -386,6 +394,7 @@
 
   onMount(async () => {
     query = getSelectedQuery()
+    schema = query.schema
```

Comment:
On line 100 we have the following code:

```js
$: hasSchema =
    Object.keys(schema || {}).length !== 0 ||
    Object.keys(query?.schema || {}).length !== 0
```

because the schema is now set in the onMount, are we able to simplify that to:

```js
$: hasSchema = Object.keys(schema || {}).length !== 0
```